src/server/enrichment-map/gene-validator/index.js

```diff
@@ -19,6 +21,19 @@ const gConvertURL = 'http://biit.cs.ut.ee/gprofiler/gconvert.cgi';
 const validatorGconvert = (query, userOptions) => {
   const promise = new Promise((resolve, reject) => {
     const formData = _.assign({}, defaultOptions, userOptions, { query: query });
+    formData.organism =  formData.organism.toLowerCase();
+    formData.target = formData.target.toUpperCase();
+    const invalidInfo = {};
+    if (!validOrganism.includes(formData.organism)) {
+      invalidInfo.invalidOrganism = formData.organism;
+    }
+    if (!validTarget.includes(formData.target)) {
+      invalidInfo.invaldTarget = formData.target;
+    }
+    if (!_.isEmpty(invalidInfo)) {
+      resolve(invalidInfo);
```

Comment:
If you're detecting an error, you should be `reject()`ing with an `Error` object.  You can subclass `Error` (e.g. `GConvertError`) if you need to include custom properties.